packages/react-native-reanimated/Common/cpp/reanimated/RuntimeDecorators/REAWorkletRuntimeDecorator.cpp

```diff
@@ -0,0 +1,14 @@
+#include "REAWorkletRuntimeDecorator.h"
+#include "JSISerializer.h"
+#include "PlatformLogger.h"
+#include "ReanimatedJSIUtils.h"
+#include "WorkletRuntime.h"
+
+namespace reanimated {
+void REAWorkletRuntimeDecorator::decorate(jsi::Runtime &rt) {
+  jsi_utils::installJsiFunction(
+      rt, "_log", [](jsi::Runtime &rt, const jsi::Value &value) {
+        PlatformLogger::log(stringifyJSIValue(rt, value));
+      });
```

Comment:
Personally I think that `_log` function should be moved to `worklets` since it can be used without Reanimated.

On Android, the tag should be taken from `workletRuntime->name_`.

Reply:
I will do it in the follow-up PR. I don't want to have any non-cpp code separated in this PR.